### PR TITLE
Add rule for preventing multiple lines in conditionals

### PR DESCRIPTION
**Pull request by jimjenkins5**
Rule prevents multiple lines in the parenthesis section of
if, while, do while, for, for in statements.

### CODE REVIEW

**Comment by yokuze** [comment]
@jthomerson Reviewed. This looks good.

File: lib/rules/no-multiline-conditionals.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+/**
+* Prevents multiple lines in within the parenthesis of if/else if/for/for in/while/do while statements.
```
Comment (jthomerson) [inline lib/rules/no-multiline-conditionals.js]: `s/in //`

Comment (jthomerson) [inline lib/rules/no-multiline-conditionals.js]: Also, `s/parenthesis/parentheses/`

**Comment by jthomerson** [comment]
@jimjenkins5 just two little notes about grammatical issues and then I can merge. Thanks!

**Pull request merged**